### PR TITLE
Update notebooks for 24.3 release and add new notebooks

### PR DESCRIPTION
**Pull request by aliseyfi75**

### CODE REVIEW

**Comment by YashaPushak** [comment]
https://github.com/oracle-samples/automlx?tab=readme-ov-file#examples
This section of the readme needs updating to reflect the new notebooks.

**Comment by YashaPushak** [comment]
https://github.com/oracle-samples/automlx?tab=readme-ov-file#license
Copyright is out of date

**Comment by YashaPushak** [comment]
https://github.com/oracle-samples/automlx?tab=readme-ov-file#documentation
docs.oracle.com embedded URL is stale

**Comment by YashaPushak** [comment]
https://github.com/oracle-samples/automlx/blob/main/LICENSE.txt
Copyright is stale here too

File: LICENSE.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2023 Oracle and/or its affiliates.
```
Comment (YashaPushak) [inline LICENSE.txt:1]: Should be 2023-2024 :)

File: README.md [hunk 2]
```diff
@@ -15,16 +15,21 @@ To run the demo notebooks yourself:
 
 ## Documentation
 
-The demo notebooks in this repository serve as supplementary documentation for the AutoMLx package. The AutoMLx class documentation is available on [docs.oracle.com](https://docs.oracle.com/en-us/iaas/tools/automlx/latest/html/multiversion/v23.1.1/index.html).
+The demo notebooks in this repository serve as supplementary documentation for the AutoMLx package. The AutoMLx class documentation is available on [docs.oracle.com](https://docs.oracle.com/en-us/iaas/tools/automlx/latest/v24.3.0/index.html).
```
Comment (YashaPushak) [inline README.md:18]: You can link to https://docs.oracle.com/en-us/iaas/tools/automlx/latest/latest/ instead

File: README.md [hunk 3]
```diff
@@ -40,7 +45,7 @@ Please consult the [security guide](./SECURITY.md) for our responsible security
 
 ## License
 
-Copyright (c) 2023 Oracle and/or its affiliates.
+Copyright (c) 2024 Oracle and/or its affiliates.
```
Comment (YashaPushak) [inline README.md:48]: Same here

File: README.md [hunk 4]
```diff
@@ -60,3 +65,5 @@ The AutoMLx demo notebooks download and use several third-party datasets to show
...
 | California Housing | No License                                                                                                               | This dataset was derived from the 1990 U.S. census, using one row per census block group. The target variable is the median house value for California districts.                                                     |
 | Newsgroup 20       | No License                                                                                                               | The 20 Newsgroups data set is a collection of approximately 20,000 newsgroup documents, partitioned (nearly) evenly across 20 different newsgroups.                                                                   |
+| PneumoniaMNIST     | [CC By 4.0](https://creativecommons.org/licenses/by/4.0/legalcode)                                                       | The PneumoniaMNIST is consist of 5,856 pediatric chest X-Ray images.                                                                   |
```
Comment (YashaPushak) [inline README.md:68]: What about newsgroup20?

Comment (aliseyfi75) [reply README.md:68]: It's the dataset above this one.

**Review by ritahuja — DISMISSED** [review]
Reviewed the html contents. All looks good.

**Review by YashaPushak — APPROVED** [review]

**Review by ritahuja — APPROVED** [review]
lgtm

**Pull request merged**